tests/test_bot.py

```diff
@@ -320,6 +321,18 @@ async def test_log_decorator(self, bot, caplog):
             assert caplog.records[0].getMessage().startswith("Entering: get_me")
             assert caplog.records[-1].getMessage().startswith("Exiting: get_me")
 
+    @bot_methods(ext_bot=False)
+    def test_api_methods_have_log_decorator(self, bot_class, bot_method_name, bot_method):
+        """Check that all bot methods have the log decorator ..."""
+        # not islower() skips the camelcase aliases
+        if not bot_method_name.islower():
+            return
+        source = "".join(inspect.getsourcelines(bot_method)[0])
```

Comment:
we could use `getsource` instead, has the same result

Reply:
nice catch :)